superset/assets/src/dashboard/util/findFirstParentContainer.js

```diff
@@ -0,0 +1,19 @@
+import { TABS_TYPE } from './componentTypes';
```

Comment:
we should write a simple test for this, or come back to it when we git users testing things

Reply:
added test.